tests/test_aliases.py

```diff
@@ -176,3 +176,20 @@ class Config(BaseConfig):
 
     assert DataClass(x=123).to_dict() == {"alias": 123}
     assert DataClass(x=None).to_dict() == {"alias": None}
+
+
+def test_by_field_with_loose_deserialize():
+    @dataclass
+    class DataClass(DataClassDictMixin):
+        a: int = field(metadata={"alias": "alias_a"})
+        b: Optional[int] = field(metadata={"alias": "alias_b"})
```

Comment:
To increase coverage it's worth adding a field with a default value and a field of type with passing through strategy such as `str`.

Reply:
Would changing `b` to `Optional[str]` suffice?

Reply:
I can check it out in a week when I get back from vacation. In the meantime you can check it by running pytest with coverage report:
```
pip install -e .
pip install -r requirements-dev.txt
pytest --cov-report html --cov=. tests
```

Reply:
I tied into the code builder tests in data type tests to leverage the automation you've already built. Relevant changes in `builder.py` have 100% coverage now.